inputs/simple-list.vue

```diff
@@ -152,6 +153,15 @@
       };
     },
     computed: {
+      autocompleteOpts() {
+        let opts = this.args.autocomplete;
+
+        if (_.has(this.args, 'modifyList') && _.has(this.args.modifyList, 'allowRemove')) {
+          _.set(opts, 'allowRemove', this.args.modifyList.allowRemove);
```

Comment:
by having the args look like (something like):

```
autocomplete:
  list: some-list
  allowRemove: true
```

this whole part of the code (and the api in general) would be a lot simpler